README.rst

```diff
@@ -53,6 +53,35 @@ Now write a ``jenkins.yml`` file and open a PR::
 Many instructions are available. Just ask the bot by commenting ``jenkins:
 help`` in an open PR!
 
+The bot's story
+===============
+
+The bot executes the extensions listed in setup.py's entry point
```

Comment:
Par exemple:

*The bot loads extensions from the setuptools entrypoint `jenkins_epo.bot.extensions`. Each extension has a `stage` key to order extensions execution. The `jenkins-epo list-extensions` command outputs the pipeline.*